server.js

```diff
@@ -6217,6 +6217,45 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 
+// Buildkite integration.
+camp.route(/^\/buildkite\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var identifier = match[1];  // eg, 3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489
+  var branch = match[2];  // eg, master
+  var format = match[3];
+
+  var url = 'https://badge.buildkite.com/' + identifier + '.json?branch=' + branch;
+  var badgeData = getBadgeData('buildkite', data);
+
+  request(url, function(err, res, buffer) {
+    if (err != null) {
```

Comment:
Could you update this to use the [`checkErrorResponse()`](https://github.com/badges/shields/blob/master/lib/error-helper.js#L8-L24) helper here.
[Example usage](https://github.com/badges/shields/blob/master/server.js#L569-L572)

**Note:**
_if you want to change the default `not found` message on `404 not found` errors, you can add an extra (string) parameter to the end which will be returned instead_